bookwyrm/forms.py

```diff
@@ -242,14 +243,39 @@ def clean(self):
         filetype = cleaned_data.get("filetype")
         book = cleaned_data.get("book")
         domain = urlparse(url).netloc
+
+        try:
+            response = requests.head(url, timeout=3)
+
+            if response.status_code in range(300, 308):
```

Comment:
I believe you can pass `allow_redirects=True` to the original request instead of checking for redirect status codes